SaltSquareIO/sports-app-react-fe
# Registration page UI and Navigation between login and registration

[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0g7kc2)
[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0g7knv)

## Summary of change

This PR implements a registration form with the following elements and functionality:

- A first name input field.
- A last name input field.
- An email input field.
- A password input field where the input text is hidden.
- A button to toggle the password visibility (revel/hide password).
- A "Register" button that logs the first name, last name, email and password in the console upon clicking.
- Below the Register button, a link with the text "Already have an account? Log in here!" Clicking the link navigates the user to the login page (BASE_URL).

This PR also covers the new changes made to the login page:

- Below the Login button, a link with the text "New user? Register here!". Clicking the link navigates the user to the registration page (BASE_URL/register).

## QA Notes

1. Navigate to BASE_URL/register.
2. Ensure that the registration form is visible.
3. Input values in all fields.
4. While entering a password in the password input field, make sure the characters are hidden.
5. Toggle the password visibility using the eye icon button. It should revel/hide the password text.
7. Check the console to confirm that the entered values are printed out. 
8. Below the "Register" button, click "Already have an account? Log in here!" to ensure it redirects to the login page.
9. On the login page, input your email and password.
10. Check the console to confirm that the entered values are printed out.
11. Below the "Login" button, click "New user? Register here!" to ensure it redirects to the registration page.
